examples/optim.jl

```diff
@@ -0,0 +1,219 @@
+# # Optimizing agent-based models
+
+# Agent-based models (ABMs) are computationally more expensive than analytical models, and can have many parameters. Sometimes we need to fine-tune a model's parameters to a specific outcome. Brute-force algorithms can take too long for testing each parameter setting. Even if it was feasbile to run the model for every parameter setting, it would not be enough because ABMs are stochastic and the effect of a parameter setting should be derived from running the model several times and taking its average behavior.
```

Comment:
> Agent-based models (ABMs) are computationally more expensive than analytical models

This sentence is straight-out false, since it is **absolute**, and for example weather models are also "analytical". But more importantly, what is the point of this sentence? Why is there a need to compare ABMs with analytical models here?

Reply:
An adverb, such as "often", would solve the objection. The sentence hints at a reason why ABM parameter spaces are more difficult explore and optimize. But now if you and Tim don't like the sentence, as he as already excluded it, I don't mind not having it there.